earthpy/tests/test_plotrgb.py

```diff
@@ -0,0 +1,101 @@
+"""Tests for plot_rgb function"""
+
+import numpy as np
+import numpy.ma as ma
+import matplotlib.pyplot as plt
+import pytest
+import rasterio as rio
+from earthpy.plot import plot_rgb
+from earthpy.io import path_to_example
+
+plt.show = lambda: None
+
+
+@pytest.fixture
+def rgb_image():
+    """Fixture holding an RGB image for plotting"""
+    with rio.open(path_to_example("rmnp-rgb.tif")) as src:
+        rgb = src.read()
+    return rgb
+
+
+def test_1band(rgb_image):
+    """Test to ensure the input image has 3 bands to support rgb plotting"""
+    rgb_1band = rgb_image[1]
+
+    with pytest.raises(Exception):
+        plot_rgb(rgb_1band)
+
+
+def test_ax_provided(rgb_image):
+    """Test to ensure the plot works when an explicit axis is provided"""
+    f, ax1 = plt.subplots()
+    f, ax = plot_rgb(rgb_image, ax=ax1)
+
+    rgb_im_shape = rgb_image.transpose([1, 2, 0]).shape
+    the_plot_im_shape = ax.get_images()[0].get_array().shape
+    assert rgb_im_shape == the_plot_im_shape
+    plt.close(f)
+
+
+def test_ax_not_provided(rgb_image):
+    """Test to ensure that plot_rgb produces a output image on an axis
```

Comment:
ok so this is the docstring here 

`    """Test to ensure that plot_rgb produces a output image on an axis
    when an axis object is not provided."""`

i don't think we want to add more lines here. so are you suggesting we go through all docstrings and shorten them for the tests? in this case a user wouldn't see them but we will . thoughts?

Reply:
I was asking whether you wanted to have folks adhere to PEP 257 for multi-line docstrings, e.g.: 

```python
def test_ax_not_provided(rgb_image):
    """Test plot_rgb without user-provided axis

    plot_rgb should produce an axis when an axis object is not provided.
    """
```

If not, that's fine. Rasterio seems not to have docstrings for most of their tests, which I think is fine if the test is well-named and easy to understand, but as we start to have more folks contributing tests it could be nice to be able to point to a PEP to provide guidance, particularly if the tests are not well-named and/or hard to follow without extra narrative in a docstring.

Reply:
Ok wonderful. I have started a new issue associated with this comment to ensure it doesn't get lost. but let's implement this in a separate PR @mbjoseph   as we just discussed :)